src/components/ListingCard/styles.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+// styled components
+export const CardBody = styled.div<{ $selected?: boolean }>`
+  display: flex;
+  flex-direction: column;
+  border: 1px solid lightgray;
+  padding: 1rem;
+  border-radius: 10px;
+  transition: 150ms;
+  cursor: pointer;
+  gap: 1rem;
+
+  ${({ $selected }) => $selected && `border-color: #097A62`};
+
+  &:hover {
+    box-shadow: 0 4px 4px 1px rgba(0, 0, 0, 0.1);
+    transform: translateY(-2px);
+  }
+`;
+
+export const CardTitle = styled.h2`
+  font-size: 1.5rem;
+  margin: 0;
+`;
+
+export const TagRow = styled.div`
+  display: flex;
+  gap: 1rem;
+`;
+
+export const CardTag = styled.span<{ color: string }>`
```

Comment:
Just looked through your commit history, I think moving this here makes sense given that right now we don't have this tag anywhere outside of the case cards. Nice!

Reply:
Also it allows for the `styles.tsx` split to work well :))